src/custom_utils/adf.py

```diff
@@ -3,13 +3,62 @@
 import json
 
 
+DEFAULT_CONFIG_PARAMETER = '{"EXAMPLE":{"type":"adsl","dataset":"EXAMPLE","container":"EXAMPLE"}}'
+
+
 def get_parameter(dbutils, parameter_name, default_value=''):
     """Creates a text widget and gets parameter value. If ran from ADF, the value is taken from there."""
     dbutils.widgets.text(parameter_name, default_value)
     return dbutils.widgets.get(parameter_name)
 
 
-def get_parameter_json(dbutils, parameter_name):
+def get_config_parameter(dbutils, parameter_name):
     """Gets parameter and converts it to a dict."""
-    json_str_config = get_parameter(dbutils, parameter_name, '{"": ""}')
+    json_str_config = get_parameter(dbutils, parameter_name, DEFAULT_CONFIG_PARAMETER)
     return json.loads(json_str_config)
+
+
+def get_source_config(dbutils):
+    """Gets the source configuration and verifies it."""
+    source_config = get_config_parameter(dbutils, 'SourceConfig')
+    _verify_config(source_config)
+
+    return source_config
+
+
+def get_destination_config(dbutils):
+    """Gets the destination configuration and verifies it."""
+    destination_config = get_config_parameter(dbutils, 'DestinationConfig')
+
+    if len(destination_config) > 1:
+        raise Exception('You are only allowed to have one destination dataset.')
+
+    _verify_config(destination_config)
+
+    return destination_config
+
+
+def _verify_config(config):
+    """Runs through the dataset configs in a source/destination config and checks the schema.
+
+    :param config:  An example could be {"<dataset_identifier>": {"type":"adsl","dataset":"<dataset_name>","container":"landing"}}
+    """
+    if 'EXAMPLE' in config:
+        raise Exception('Your are using an example config. If developing in Databricks, fill out the text widget.')
```

Comment:
For at maksimere brugervenligheden, skal vi så ikke skrive:
`'It looks like you are using an example configuration.\nIf developing in Databricks, manually fill out the source- and destination-configuration widgets and run the notebook again.'`